### PR TITLE
Include the required units of the Forchheimer IPR model

### PR DESCRIPTION
**Pull request by caio-ramos**
ASIM-5275

Adds specific units employed in the Forchheimer IPR model. A new category and quantity type is defined for linear and quadratic parameters.

### CODE REVIEW

**Review by nicoddemus — CHANGES_REQUESTED** [review]
Nice!

Please update the CHANGELOG with the new units. :+1:

File: src/barril/units/posc.py [hunk 1]
```diff
@@ -163,6 +163,16 @@ def FillUnitDatabaseWithPosc(
     db.AddUnitBase("specific volume", "cubic metres/kilogram", "m3/kg")
     db.AddUnitBase("molar volume", "cubic metres/mole", "m3/mol")
     db.AddUnitBase("productivity index", "cubic metres/second pascal", "m3/Pa.s")
+    db.AddUnitBase(
+        "forchheimer linear productivity index",
+        "square pascal/standard cubic metres per second",
+        "Pa2/scm.s",
```
[the remark this answers is not in the document]
Comment (caio-ramos) [reply src/barril/units/posc.py]: You're 100% right. The corrections have been made.

**Review by nicoddemus — APPROVED** [review]

File: src/barril/units/_tests/test_posc.py [hunk 2]
```diff
@@ -485,6 +485,24 @@ def testProductivityIndex() -> None:
     assert default.GetValue("m3/d/kgf/cm2") == approx(8472945600.38827)
 
 
+def testForchheimerLinearCoefficient() -> None:
+    default = units.Scalar("forchheimer linear productivity index", 1, "Pa2.s/scm")
+    assert default.value == 1.0
+    assert default.GetValue("Pa2.d/scm") == approx(1.1574074074074073e-05)
```
[the remark this answers is not in the document]
Comment (ggrbill) [reply src/barril/units/_tests/test_posc.py:491]: `sm6` also sounds awkward to me. 
Let's use `scm2`. It looks simpler and cleaner.

**Review by ggrbill — APPROVED** [review]
Good Job!

**Review by Gualcar — APPROVED** [review]

**Pull request closed without merging**